pkg/gardenadm/cmd/init/init.go

```diff
@@ -199,6 +206,12 @@ func run(ctx context.Context, opts *Options) error {
 			SkipIf:       podNetworkAvailable,
 			Dependencies: flow.NewTaskIDs(deployGardenerResourceManagerIntoPodNetwork),
 		})
+		getGardenerResourceManagerServiceIP = g.Add(flow.Task{
+			Name:         "Get gardener-resource-manager service IP",
+			Fn:           b.GetAddGardenerResourceManagerServiceIP,
+			SkipIf:       podNetworkAvailable,
+			Dependencies: flow.NewTaskIDs(waitUntilGardenerResourceManagerInPodNetworkReady),
+		})
```

Comment:
I do not think that `SkipIf: podNetworkAvailable` is correct here.
We do not want some other steps to be duplicated unnecessarily, but this one is not duplicated at all and should always run.
WDYT?